packages/flutter_tools/lib/src/isolated/resident_web_runner.dart

```diff
@@ -393,7 +393,7 @@ Please provide a valid TCP port (an integer between 0 and 65535, inclusive).
     }
   }
 
-  WebCompilerConfig get _compilerConfig => (debuggingOptions.webUseWasm)
```

Comment:
This getter violates the [style guide](https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#consider-using--for-short-functions-and-methods).

It's not your fault of course, but instead of just removing the parentheses here, I think it'd be good to convert to a block body.

```dart
  WebCompilerConfig get _compilerConfig {
    if (debuggingOptions.webUseWasm) {
      return WasmCompilerConfig(
        optimizationLevel: 0,
        stripWasm: false,
        renderer: debuggingOptions.webRenderer
      );
    }
    return JsCompilerConfig.run(
      nativeNullAssertions: debuggingOptions.nativeNullAssertions,
      renderer: debuggingOptions.webRenderer,
    );
  }
```